packages/components/Menu/src/MenuGroup/MenuGroup.tsx

```diff
@@ -39,7 +44,21 @@ export const MenuGroup = compose<MenuGroupType>({
         }
         return child;
       });
-      return <Slots.root {...mergedProps}>{childrenWithSet}</Slots.root>;
+
+      return (
+        <Slots.root {...mergedProps}>
+          <Slots.focusZone
+            // avoid error that fires when props are passed into React.fragment
+            {...(hasFocusZone && {
+              focusZoneDirection: 'vertical',
+              enableFocusRing: false,
+              isCircularNavigation: true,
```

Comment:
Does this make it so when you reach the end of the menu group, down arrow takes you to the first menu item in the current menu group? If so, this doesn't align with the native behavior on win32. Up/down arrows should allow you to move to every item in the menu, even between groups, but maybe I'm misunderstanding what this does and it will allow you to do that.

Reply:
oh ok. I misunderstood the spec then. Yeah this will make it so that hitting the end of the group will bring you back to the beginning. I thought that's what the bug was? We'll take this offline